### PR TITLE
Use custom easyblocks in 2023.

### PR DESCRIPTION
**Pull request by bartoldeman**
There are checked out at
/cvmfs/soft.computecanada.ca/easybuild/site-packages/custom-easyblocks/2023/easyblocks and correspond to the 2023 branch of the CC easybuild-easyblocks repository, in a similar fashion as
https://github.com/easybuilders/JSC/tree/2024/Custom_EasyBlocks

### CODE REVIEW

**Comment by mboisson** [comment]
So, with that scheme, what will be the way to keep modified easyblocks in sync with upstream ones when upstream ones get updates ? Is this a fork of the upstream repository from which we remove everything that is not modified ?

**Comment by mboisson** [comment]
Basically, I think the custom branch should contain our version of these files: 
```
git diff computecanada-main upstream_4.7.x --name-only
easybuild/easyblocks/a/abaqusx.py
easybuild/easyblocks/a/amber.py
easybuild/easyblocks/b/boost.py
easybuild/easyblocks/c/cgal.py
easybuild/easyblocks/c/cst.py
easybuild/easyblocks/d/demon2k.py
easybuild/easyblocks/e/esmf.py
easybuild/easyblocks/f/feko.py
easybuild/easyblocks/f/fftw.py
easybuild/easyblocks/f/flexiblas.py
easybuild/easyblocks/f/fsl.py
easybuild/easyblocks/g/gromacs.py
easybuild/easyblocks/generic/apptainer.py
easybuild/easyblocks/generic/bundle.py
easybuild/easyblocks/generic/cmakemake.py
easybuild/easyblocks/generic/intelbase.py
easybuild/easyblocks/generic/mesonninja.py
easybuild/easyblocks/generic/nix.py
easybuild/easyblocks/generic/perlmodule.py
easybuild/easyblocks/generic/pythonpackage.py
easybuild/easyblocks/generic/rubygem.py
easybuild/easyblocks/generic/systemcompiler.py
easybuild/easyblocks/h/hdf5.py
easybuild/easyblocks/j/julia.py
easybuild/easyblocks/l/lammps.py
easybuild/easyblocks/l/lammps_cc.py
easybuild/easyblocks/l/lumerical.py
easybuild/easyblocks/m/materialsstudio.py
easybuild/easyblocks/m/mcr.py
easybuild/easyblocks/m/mrbayes.py
easybuild/easyblocks/n/namd.py
easybuild/easyblocks/n/ncl.py
easybuild/easyblocks/n/neuron.py
easybuild/easyblocks/o/oasyslsdyna.py
easybuild/easyblocks/o/openbabel.py
easybuild/easyblocks/o/openfoam.py
easybuild/easyblocks/p/perl.py
easybuild/easyblocks/p/petsc.py
easybuild/easyblocks/p/pgi.py
easybuild/easyblocks/p/picard.py
easybuild/easyblocks/p/pwrf.py
easybuild/easyblocks/q/quantumespresso.py
easybuild/easyblocks/r/rosetta.py
easybuild/easyblocks/s/samtools.py
easybuild/easyblocks/s/siesta.py
easybuild/easyblocks/s/starccm.py
easybuild/easyblocks/s/superlu.py
easybuild/easyblocks/t/tau.py
easybuild/easyblocks/t/tbb.py
easybuild/easyblocks/t/tensorrt.py
easybuild/easyblocks/t/torch.py
easybuild/easyblocks/t/trilinos.py
easybuild/easyblocks/t/trinity.py
easybuild/easyblocks/u/ucx_plugins.py
easybuild/easyblocks/v/visit.py
easybuild/easyblocks/x/xcrysden.py
```

File: 2023/config.cfg [hunk 1]
```diff
@@ -4,3 +4,4 @@ allow-loaded-modules=gentoo
 cuda-compute-capabilities = 6.0,7.0,7.5,8.0
 sysroot=/cvmfs/soft.computecanada.ca/gentoo/2023/x86-64-v3
 env-for-shebang = /cvmfs/soft.computecanada.ca/gentoo/2023/x86-64-v3/usr/bin/env -S
+include-easyblocks = /cvmfs/soft.computecanada.ca/easybuild/site-packages/custom-easyblocks/2023/easyblocks/*.py,/cvmfs/soft.computecanada.ca/easybuild/site-packages/custom-easyblocks/2023/easyblocks/generic/*.py
```
[the remark this answers is not in the document]
Comment (bartoldeman) [reply 2023/config.cfg]: corrected now (with extra easybuild directory)

**Comment by mboisson** [comment]
lgtm

**Pull request merged**